shell-scripts/copy-vendor.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const shell = require("shelljs");
```

Comment:
Node has built-in `fs` module. Could you use it instead of this dependency?

Reply:
I just did, only this waiting for you to answer the question above for push.